src/components/campaigns/Summary/Stat/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+const content = (loading, number, label) => {
+  if (loading) return <span />
+  return (
+    <div>
+      <span className='CampaignSummaryStat__number' key='number'>
+        {`${number} `}
```

Comment:
Just spitballing and not required to change, but you might be able to get away with:

```js
{number}&nbsp;
```